.tractusx

```diff
@@ -0,0 +1,22 @@
+###############################################################
+# Copyright (c) 2022, 2023 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License, Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0.
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+###############################################################
+
+product: "Tractus-X Knowledge Agents AAS Bridges (KA-AAS)"
```

Comment:
Are all of the Knowledge Agent related repos separate products?
Or could we think about combining them all to a "Tractus-X Knowledge Agents" product.
The only difference would be, that the repositories would be grouped on our [Release Guideline checks dashboard](https://eclipse-tractusx.github.io/sig-release/). You can have a look at Portal for example.

Reply:
As a product, there are one.
But there is no "leading" repo (the KIT under [eclipse-tractusx.github.io](https://eclipse-tractusx.github.io/docs-kits/next/category/agents-kit) is the real glue ...)
.... although with the CX 24.03 we will revamp the architecture ... then knowledge-agents will be leading repo. I will change the .tractusx files accordingly, correct?

Reply:
see also https://github.com/eclipse-tractusx/knowledge-agents-edc/pull/127 and https://github.com/eclipse-tractusx/knowledge-agents/pull/52